colcon_ros_gradle/package_identification/ros_gradle.py

```diff
@@ -1,18 +1,18 @@
 # Copyright 2018 Esteve Fernandez
 # Licensed under the Apache License, Version 2.0
 
-import os
-from pathlib import Path
-import re
-
-from colcon_core.package_identification import logger
 from colcon_core.package_identification \
     import PackageIdentificationExtensionPoint
 from colcon_core.plugin_system import satisfies_version
-
+from colcon_gradle.package_identification.gradle \
+    import GradlePackageIdentification
+from colcon_ros.package_identification.ros \
+    import RosPackageIdentification
 
 class RosGradlePackageIdentification(PackageIdentificationExtensionPoint):
-    """Identify Gradle packages with `build.gradle` files."""
+    """
+    Identify ROS Gradle packages with `package.xml` and `build.gradle` files.
+    """
 
     PRIORITY = 160
```

Comment:
I would highly recommend to add a comment here to document why this priority value was chosen.